features/facebookImport/src/model/entities/utils/json-serialization.js

```diff
@@ -0,0 +1,13 @@
+export function toJsonObject(targetObject) {
+    let jsonObject = {};
+    Object.assign(jsonObject, targetObject);
+    jsonObject.__class__ = targetObject.constructor.name;
```

Comment:
Nice hack here. But. If we're being picky about classes, maybe the whole entities thing should be in TypeScript, which it can be.
Also, there's a whole framework for serialization/deserialization in the polypod, `bubblewrap`. So again another reason why we should *really* think what's the place for this within the whole picture.